torchci/lib/bot/mergeBot.ts

```diff
@@ -2,24 +2,38 @@ import { Probot } from "probot";
 import { addComment, reactOnComment } from "./botUtils";
 
 function mergeBot(app: Probot): void {
-  const mergeCmdPat = new RegExp("^\\s*@pytorch(merge|)bot\\s+(force\\s+)?merge\\s+this");
+  const mergeCmdPat = new RegExp(
+    "^\\s*@pytorch(merge|)bot\\s+(force\\s+)?merge\\s+this"
+  );
   const revertCmdPat = new RegExp("^\\s*@pytorch(merge|)bot\\s+revert\\s+this");
-  const rebaseCmdPat = new RegExp("^\\s*@pytorch(merge|)bot\\s+rebase\\s+(me|this)");
+  const rebaseCmdPat = new RegExp(
+    "^\\s*@pytorch(merge|)bot\\s+rebase\\s+(me|this)"
+  );
+  const mergeOnGreenCmdPat = new RegExp(
+    "^\\s*@pytorch(merge|)bot\\s+land\\s+(me|this)"
```

Comment:
I guess we can merge this into the merge command. I thought it would be easier to have a different command for merging on green since 
1. we can just remove the block when we're ready
2. have the commands have the same format without any flags and such

Reply:
Yea, it doesn't ultimately matter but I'd err towards what makes the most sense to users. Since they suggested ` `merge on green`, I'd just default to that.